app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MapFragment.java

```diff
@@ -161,17 +195,18 @@ public void onLocationResult(@Nullable Location location) {
                             .bearing(0)
                             .tilt(0)
                             .build();
-                    map.moveCamera(CameraUpdateFactory.newCameraPosition(currentPosition));
+                    if (position == null){
+                        map.moveCamera(CameraUpdateFactory.newCameraPosition(currentPosition));
+                }else{
```

Comment:
It's also pretty clear you didn't `ctrl+alt+l` here

Reply:
Lol